[2024-04-19T08:35:23.080+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-19T08:35:23.117+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingest_exploration_dag_v1.finish_task scheduled__2024-04-17T00:00:00+00:00 [queued]>
[2024-04-19T08:35:23.126+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingest_exploration_dag_v1.finish_task scheduled__2024-04-17T00:00:00+00:00 [queued]>
[2024-04-19T08:35:23.127+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-19T08:35:23.142+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): finish_task> on 2024-04-17 00:00:00+00:00
[2024-04-19T08:35:23.159+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=166) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-19T08:35:23.161+0000] {standard_task_runner.py:63} INFO - Started process 168 to run task
[2024-04-19T08:35:23.160+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_ingest_exploration_dag_v1', 'finish_task', 'scheduled__2024-04-17T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/data_ingest_exploration_dag.py', '--cfg-path', '/tmp/tmpo_2fan0d']
[2024-04-19T08:35:23.163+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask finish_task
[2024-04-19T08:35:23.224+0000] {task_command.py:426} INFO - Running <TaskInstance: data_ingest_exploration_dag_v1.finish_task scheduled__2024-04-17T00:00:00+00:00 [running]> on host b44d208cdeda
[2024-04-19T08:35:23.312+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ika' AIRFLOW_CTX_DAG_ID='data_ingest_exploration_dag_v1' AIRFLOW_CTX_TASK_ID='finish_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-17T00:00:00+00:00'
[2024-04-19T08:35:23.314+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-19T08:35:23.331+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-19T08:35:23.334+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'finish extract data!'"]
[2024-04-19T08:35:23.355+0000] {subprocess.py:86} INFO - Output:
[2024-04-19T08:35:23.365+0000] {subprocess.py:93} INFO - finish extract data!
[2024-04-19T08:35:23.366+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-19T08:35:23.368+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-19T08:35:23.396+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=data_ingest_exploration_dag_v1, task_id=finish_task, execution_date=20240417T000000, start_date=20240419T083523, end_date=20240419T083523
[2024-04-19T08:35:23.457+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-19T08:35:23.477+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-19T08:35:23.480+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
